examples/collections/tiger.py

```diff
@@ -10,8 +10,9 @@
 from vispy.util import load_data_file
 from vispy.util.svg import Document
 from vispy.visuals.collections import PathCollection, PolygonCollection
-from panzoom import PanZoom
-
+from panzoom import PanZoomTransform
+import faulthandler
+faulthandler.enable()
```

Comment:
what is that? to debug segfaults? I suppose you didn't intend to keep that.

FWI I also had some segfaults with this example, they might be related to the *triangles* library..?